.gitignore

```diff
@@ -3,3 +3,4 @@
 *.int
 *.lk
 *.zip
+test.MD
```

Comment:
- [ ] great job creating the .gitignore file, kindly fix the typo it should be `test.md` and not `test.MD` and kindly add `node_modules` as part of the requirements